AppConnectSwift/CreateAccountViewController.swift

```diff
@@ -16,8 +16,8 @@ class CreateAccountViewController: UIViewController {
     }
     
     @IBAction func createAccount(sender: AnyObject) {
-        let clientFactory = MDClientFactory.sharedInstance()
-        let client = clientFactory.clientOfType(MDClientType.Network);
+        let client = MDClientFactory.sharedInstance().clientOfType(MDClientType.Network);
```

Comment:
I think this was overwritten by the conflicts - this should be `MDClientType.Hybrid`